app/views/publish/courses/a_level_requirements/add_a_level_to_a_list/new.html.erb

```diff
@@ -0,0 +1,60 @@
+<% content_for :page_title, title_with_error_prefix(t("course.#{@wizard.current_step.model_name.i18n_key}.heading"), @wizard.current_step.errors && @wizard.current_step.errors.any?) %>
+
+<% content_for :before_content do %>
+  <% if @wizard.current_step.maximum_number_of_a_level_subjects? %>
+    <%= govuk_back_link_to(
+        publish_provider_recruitment_cycle_course_a_levels_are_any_a_levels_required_for_this_course_path(
+          @provider.provider_code,
+          @provider.recruitment_cycle_year,
+          @course.course_code
+        )
+      ) %>
+  <% else %>
+    <%= govuk_back_link_to(
+        publish_provider_recruitment_cycle_course_a_levels_what_a_level_is_required_path(
+          @provider.provider_code,
+          @provider.recruitment_cycle_year,
+          @course.course_code
+        )
+      ) %>
+  <% end %>
+<% end %>
```

Comment:
I am adding/editing another card to handle another scenario when there is between 1 up to 3 A levels and the user clicks back. For now if they click "Back" is like the same as answering yes and anothing another A level (up to 4 maximum). This is the card https://trello.com/c/mEPXi2RG/1868-a-levels-edge-case-back-link-when-user-is-adding-a-levels?filter=label:Epic%3A%20TDA (I am still editing as I wrote this comment)